source/tools/testConsistency/ClientApp/postRequest.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. 
+const http = require('http');
+
+function getPostRequestPromise(options, platformName, formData) {
+    const postReqPromise = new Promise(function(resolve, reject) {
```

Comment:
A comment here would be nice explainng what is going on - esp with this long of an anonymous function :))